transform-benchmarker/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@xstream/transform-benchmarker",
```

Comment:
can you drop the `@xstream` scope?

Reply:
Sure. I updated transform-benchmarker's `package.json` file.